docs/ota-client-guide/modules/ROOT/pages/finding-unsigned-metadata.adoc

```diff
@@ -0,0 +1,25 @@
+= Find the unsigned Root and Targets metadata
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+If you want to use your own PKI you will need to know where in your local repository can you find the metadata you want to sign.
+It may be `root.json` or `targets.json` files, both are stored under `tuf/<reponame>/roles/unsigned` path.
+
+NOTE: <reponame> is the name you specified during your repository initialization with `garage-sign init`
+
+If, however, the `unsigned/` folder is empty, most likely it means you have to do a pulling first:
+
+* Use `garage-sign root pull` to pull the unsigned `root.json`
+* Use `garage-sign targets pull` to pull the unsigned `targets.json`
+
+If you haven't previously created any targets - use `garage-sign targets init` which will create the unsigned `targets.json` as well.
```

Comment:
```suggestion
If you want to use your own PKI, you need to know where in your local repository you can find the metadata that you want to sign.
It may be `root.json` or `targets.json` files. You can find both files in the `tuf/<reponame>/roles/unsigned` folder. 

NOTE: <reponame> is the name you specified when you initialized your repository using `garage-sign init`.

If the `unsigned/` folder is empty, you need to pull the metadata files:

* To pull the unsigned `root.json` file, use `garage-sign root pull`.
* To pull the unsigned `targets.json`, use `garage-sign targets pull`.

If you have not created any targets, to create the unsigned `targets.json` file, use `garage-sign targets init`.